documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+The benefits are following:
+1. Arch Board can review the ApiView when swagger PR is created and no need to wait for SDK owners to create ApiView manually, which saves much time.
+2. Service team can change the swagger in the PR directly if Arch Board has some comments in reviewing ApiView because the swagger PR is not merged.
+
+This Document describes the process of how to onboard DPG with SDK automation pipeline. __This document is targeting for service team.__ 
+
+*If you want to go through details of pipeline implementation, please go to [Integrate DPG and ApiView in SDK Automation Pipeline](Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md).*
+
+## WorkFlow
+
+![workflow-for-service-team](workflow-service-team.png)
+
+__Description:__
+1. Creates a swagger PR with [Guide to design and creation of Data Plane REST API and Client Libraries](https://aka.ms/azsdk/dpcodegen).
+2. If the SDK was generated for this service before , please go to step 4. Otherwise, this is a new service and please go to step 3.
+3. Create branch with name `dpg/<prNumber>` in sdk repository, and add autorest configuration file. Then add a comment `/azp run` in swagger PR.
```

Comment:
Do you have the description for language specific configuration files? 
It is better to add the link (or place holder if this is not ready) on how to generate that file for each SDK.